radicle-httpd/src/api/v1/projects.rs

```diff
@@ -939,4 +1013,65 @@ mod routes {
             ])
         );
     }
+
+    #[tokio::test]
+    async fn test_projects_patches() {
+        let tmp = tempfile::tempdir().unwrap();
+        let app = super::router(test::seed(tmp.path()));
+        let response = get(&app, "/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/patches").await;
+
+        assert_eq!(response.status(), StatusCode::OK);
+        assert_eq!(
+            response.json().await,
+            json!([
+              {
+                "id": "5de9f17ca5326258412ab02f9a5339b6482198ce",
+                "author": {
+                    "id": "z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi"
+                },
+                "title": "A new `hello word`",
+                "description": "change `hello world` in README to something else",
+                "state": "proposed",
+                "target": "delegates",
+                "tags": [],
+                "revisions": [
+                    {
+                        "id": [1, "z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi"],
```

Comment:
I've merged the op-id changes and opened a pr to use them for json: https://github.com/radicle-dev/heartwood/pull/269

I'd rebase this pr on it so that you can test the correct op-id serialization.

Reply:
done, check again, now we're getting `z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi/1`